types/express-flash-notification/express-flash-notification-tests.ts

```diff
@@ -0,0 +1,57 @@
+import * as express from "express";
+import * as flash from "express-flash-notification";
```

Comment:
Since `express` and `flash` are functions, it should be `import ... = require(...)` here rather than `import * as ... from ...`

Reply:
I have changed the code to what you have specified. It should be available now.